### ISSUE TITLE
iTunes 12.6.1.25 / 10.12.5 Not Respecting Profile Settings

### ISSUE THREAD

**Issue by walkintom** [rodchristiansen/Profiles#1]
This profile sadly doesn't appear to work correctly with iTunes 12.6.1.25 on 10.12.5 - is it possible that any of the keys have been changed? I've tried locking the store via Restrictions in the app and checking the values it wrote to com.apple.iTunes.plist and they looked identical, I can't fathom why the profile itself isn't working.

**Comment by rodchristiansen** [comment]
It's working fine for me here:
![screen shot 2017-05-24 at 6 24 24 pm](https://cloud.githubusercontent.com/assets/7052526/26432126/590fde2c-40ae-11e7-980d-cdbb062c755c.png)

**Comment by walkintom** [comment]
Hi Rod

Thank you for getting back to me. That’s really frustrating, I can’t fathom what I’m missing. I’ve copied your profile raw, installed it but it just doesn’t seem to take effect. Are there any prerequisites or dependants I could be missing?

Kind regards
Tom

**Comment by rodchristiansen** [comment]
Take a look at `/Library/Managed Preferences/com.apple.iTunes.plist`, that will show you what keys are being applied by the system to that .plist based on the .mobileconfig — should help you pin point the issue.

**Comment by walkintom** [comment]
Thank you, I got to the bottom of it with your suggestion. No matter what I did, the only setting present in /Library/Managed Preferences/com.apple.iTunes.plist/ or in any user’s subfolder therein was disableAppleMusic.

I then realised Apple Music had been disabled via Mosyle’s GUI, this caused a conflict with the profile. Enabling the Apple Music setting in Mosyle allowed your profile to take effect as desired.

Thanks again for your help with this.
